config-raw.json

```diff
@@ -606,6 +606,31 @@
                             "comment": "The url where clients are redirected after they authorized Vikunja to access their Microsoft todo tasks.\nThis needs to match the url you entered when registering your Vikunja instance at Microsoft.\nWhen using the official Vikunja frontend, set this to `\u003cservice.publicurl\u003e/migrate/microsoft-todo` (the default value).\nOtherwise, set this to an url which then makes a request to /api/v1/migration/microsoft-todo/migrate\nwith the code obtained from the Microsoft Todo api."
                         }
                     ]
+                },
+                {
+                    "key": "deck",
+                    "children": [
+                        {
+                            "key": "enable",
+                            "default_value": "false",
+                            "comment": "Whether to enable the Nextcloud Deck migrator."
+                        },
+                        {
+                            "key": "clientid",
```

Comment:
But the point of the new structure is to add the nextcloud login url in the UI? Why do I need to provide a client id and secret in the config?